README_libf3d.md

```diff
@@ -195,7 +195,7 @@ Here is an example showing how to use libf3d python bindings:
 ```python
 import f3d
 
-eng = f3d.engine()
+eng = f3d.engine(f3d.window.NATIVE)
```

Comment:
is this really needed since you added the default val ?

Reply:
No, but the users will read master documentation for 1.3.1, so let's keep the argument in the documentation until the fix is released

Reply:
good point